txnkv/transaction/pipelined_flush.go

```diff
@@ -334,6 +335,18 @@ func (c *twoPhaseCommitter) buildPipelinedResolveHandler(commit bool, resolved *
 		maxBackOff = CommitSecondaryMaxBackoff
 	}
 	regionCache := c.store.GetRegionCache()
+	// the handler function runs in a different goroutine, should copy the required values before it to avoid race.
```

Comment:
As dicussed in above comments, this may not be a perfect solution, we still need to find a way to avoid race completely.

Reply:
Maybe set `txn.committer` to `nil` which disable further accessing to rollbacked committer.